pontoon/base/migrations/0105_unescape_quotes_from_android_strings_dtd_in_translation_memory.py

```diff
@@ -0,0 +1,49 @@
+# -*- coding: utf-8 -*-
+# Generated by Django 1.10.8 on 2017-10-19 21:19
+from __future__ import unicode_literals
+
+from django.db import migrations
+from django.db.models import Q, F, Func, Value
+
+
+def unescape_quotes_in_tm_entries_from_android_strings_dtd(apps, schema):
+    """
+    Problem with the synchronization of android_strings.dtd (&quot; and &apos; entities) was fixed in eb5f8d553.
+    However, TM still contains entries with broken entities which break TMX files.
+
+    Related bug: https://bugzilla.mozilla.org/show_bug.cgi?id=1390111
```

Comment:
I'd also mention https://github.com/mozilla/pontoon/commit/4537eac40f36cff404fc36bc9f7a5c4c82b520e8 and bug 1390113. Let's change this comment to:

```
Handling of different escape sequences for mobile Android strings was fixed in eb5f8d5. However,
data migration in 4537eac only covered the Translation model, leaving out TranslationMemoryEntry.
See bugs 1390111 and 1390113 for more details.
```